lib/task.js

```diff
@@ -59,7 +59,7 @@ Task.prototype.logs = function(opts, callback) {
   var optsf = {
     path: '/tasks/' + this.id + '/logs?',
     method: 'GET',
-    isStream: args.opts.follow || false,
+    isStream: (typeof(args.opts.follow) != 'undefined') ? args.opts.follow : false,
```

Comment:
How is this an improvement or different from the original?

Reply:
Agree, changing.